monitoring/alerting/rules/database_alerts.yaml

```diff
@@ -0,0 +1,212 @@
+apiVersion: 1
+groups:
+  - interval: 5m
+    name: SAS Viya Alerts
+    folder: Database Alerts
+    orgId: 1
+    rules:
+      - title: Catalog DB Connections High
+        annotations:
+          description:
+            "Checks the in-use catalog database connections > 21. The default
+            db connection pool is 22.  If it reaches the limit, the rabbitmq queues starts
+            to fill up with ready messages causing issues with Model Studio pipelines.
+
+            Click on the  URL on how to remediate the issue."
```

Comment:
Remove quotes around description.  Remove line "Click on the URL on how to remediate the issue", I suspect it pointed to an internal runbook.  I think I would remove the reference to Model Studio as well.  That is not present in every SAS Viya deployment and I suspect/assume this alert is useful outside of the Model Studio use-case.